\

```diff
@@ -0,0 +1,245 @@
+# encoding: UTF-8
+require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/file"
+require "logstash/event"
+require "logstash/json"
+require "stud/temporary"
+require "tempfile"
+require "uri"
+
+describe LogStash::Outputs::File do
+  describe "ship lots of events to a file" do
+    tmp_file = Tempfile.new('logstash-spec-output-file')
+    event_count = 10000 + rand(500)
+
+    config <<-CONFIG
+    input {
+      generator {
+        message => "hello world"
+        count => #{event_count}
+        type => "generator"
+      }
+    }
+    output {
+      file {
+        path => "#{tmp_file.path}"
+      }
+    }
+    CONFIG
+
+    agent do
+      line_num = 0
+      
+      # Now check all events for order and correctness.
+      tmp_file.each_line do |line|
+        event = LogStash::Event.new(LogStash::Json.load(line))
+        insist {event["message"]} == "hello world"
```

Comment:
This file should not exist, my mistake.. The comment should  be made on the actual diff.

Well, I agree we need to use the `expect` syntax, but inside an agent`s block the expect method isn`t available.
Case like this should be rewritten once we have a new testing strategy.

```
     Failure/Error: block.call
     NoMethodError:
       undefined method `expect' for RSpec::Core::ExampleGroup::Nested_1::Nested_1:Class
```